src/Components/FullyResponsiveBox.tsx

```diff
@@ -3,17 +3,18 @@ import { FC, useEffect, useState } from "react"
 import { useSizeAndPosition } from "Utils/Hooks/useSizeAndPosition"
 import { scale } from "proportional-scale"
 
-interface ArticleZoomGalleryResponsiveBoxProps extends BoxProps {
+export interface FullyResponsiveBoxProps extends BoxProps {
```

Comment:
Lol @dzucconi having "ResponsiveBox" and "FullyResponsiveBox" is clearly quite confusing

Reply:
How about `FullscreenBox`

Reply:
Sure. This is a totally different use-case than a normal `ResponsiveBox` (and a pretty rare one). Good call out. `FullscreenBox` makes sense to me. Any objections or better ideas?

Reply:
`ResponsiveBox` should probably be `AspectRatioBox`...

`FullscreenAspectRatioBox`?

Reply:
FullscreenBox 👍 that makes perfect sense

Reply:
Images, not necessarily: like it could be a video or iframe. But the function of it is: scale up in both dimensions, maintaining aspect ratio.